plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -160,11 +157,8 @@ public void testJoinWithNonSelectiveBuildSide()
         assertEquals(dynamicFiltersStats.getDynamicFiltersCompleted(), 1L);
 
         DynamicFilterDomainStats domainStats = getOnlyElement(dynamicFiltersStats.getDynamicFilterDomainStats());
-        assertEquals(domainStats.getSimplifiedDomain(), Domain.create(ValueSet.ofRanges(
```

Comment:
IMO previous check was more reliable. I think we know simplified domain (strict range from (0, 100)). Could we generate it and compare `toStrings`. We could modify `getExpectedDomainStatsString` method to do so.

Reply:
The earlier approach of using toString makes the tests pass irrespective of how the printing is done. I want tests to fail in future if the formatting changes.